nb-adapters/adapter-cqld4/src/main/java/io/nosqlbench/adapter/cqld4/opmappers/Cqld4CoreOpMapper.java

```diff
@@ -61,8 +61,8 @@ public OpDispenser<Cqld4BaseOp<?>> apply(NBComponent adapterC, ParsedOp op, Long
             case raw, simple, prepared, batch -> new Cqld4CqlOpMapper(adapter).apply(adapterC, op, spaceF);
             case gremlin -> new Cqld4GremlinOpMapper(adapter, target.targetFunction).apply(adapterC, op, spaceF);
             case fluent -> new Cqld4FluentGraphOpMapper(adapter, target).apply(adapterC, op, spaceF);
-            case rainbow ->
-                new CqlD4RainbowTableMapper(adapter, spaceF, target.targetFunction).apply(adapterC, op, spaceF);
+//            case rainbow ->
+//                new CqlD4RainbowTableMapper(adapter, spaceF, target.targetFunction).apply(adapterC, op, spaceF);
```

Comment:
I see you are commenting out rainbows. Are unicorns next? :-D

Reply:
We're going to add them back in together.